### PR TITLE
Initialize Question Test

### PR DESCRIPTION
**Pull request by musayann**
Initialized question test along with it's factory.

The ruby version set in #2 is causing issues while running tests, meanwhile we returned to the original version while we find the solution.

### CODE REVIEW

**Comment by bumi** [comment]
what problem did it cause? ideally add the error messages in such cases or create a trello ticket with the error message.

**Comment by musayann** [comment]
Sorry i forgot . but here is the error message

```error
Unknown ruby interpreter version (do not know how to handle): >2.3.0,<2.7.0.
```

File: test/models/recommedation_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (fsmanuel) [inline test/models/recommedation_test.rb]: @musayann Can you delete this file?

**Comment by fsmanuel** [comment]
@musayann I think you need to rebase and run `bundle install` again.

**Comment by bumi** [comment]
:+1:

**Pull request merged**